src/Equinor.ProCoSys.IPO.Query/GetInvitationsQueries/GetInvitationsForExport/GetInvitationsForExcelQueryHandler.cs

```diff
@@ -44,17 +44,16 @@ public async Task<Result<ExportDto>> Handle(GetInvitationsForExportQuery request
             var invitationIds = orderedDtos.Select(dto => dto.Id).ToList();
             var getHistoryAndParticipants = invitationIds.Count == 1;
 
-            var invitationsWithIncludes =
-                await GetInvitationsWithIncludesAsync(invitationIds, getHistoryAndParticipants, cancellationToken);
+            var invitationsWithIncludes = await GetInvitationsWithIncludesAsync(_context, invitationIds, token);
 
-            var exportInvitationDtos = CreateExportInvitationsDtosAsync(orderedDtos).Result;
+            var exportInvitationDtos = await CreateExportInvitationsDtosAsync(orderedDtos, invitationsWithIncludes);
 
             if (getHistoryAndParticipants)
             {
                 await GetHistoryForSingleInvitationAsync(
                     invitationIds.Single(),
                     exportInvitationDtos,
-                    cancellationToken);
+                    token);
```

Comment:
use name cancellationToken? to avoid confusion with other types of tokens

Reply:
Will use cancellationToken everywhere in solution then

Reply:
I suggested rename the other way since we've used token all-over-the-place for a cancellationToken

Reply:
I have already done renaming, so I will go with whichever you agree on is the best naming :)

Reply:
cancellationToken is fine for me